src/Microsoft.AspNet.Mvc.Razor/RazorView.cs

```diff
@@ -42,6 +42,9 @@ public class RazorView : IView
             IsPartial = isPartial;
         }
 
+        /// <inheritdoc />
+        public string Path => RazorPage.Path;
```

Comment:
faster w/ the testing than I :smile:
:ok: much simpler than I thought it would be.  and, since `RazorPage.Path` is a simple property, not expensive to evaluate it multiple times.